src/docs/guides/24-how-to-use-time-based-bookings/index.md

```diff
@@ -0,0 +1,131 @@
+plan that is always available. Therefore, a time-based plan is required
+for time-based listings.
+
+The availability plan is passed as parameter to the API when a listing
+is created or updated. The following code passes a time-based
+availability plan to the API using the
+[Flex JavaScript SDK](https://github.com/sharetribe/flex-sdk-js):
+
+<!-- prettier-ignore -->
+```js
+sdk.ownListings
+  .create(
+    {
+      title: 'Cyclocross bike',
+      description: 'A steady gravel grinder',
+      geolocation: new LatLng(40.64542, -74.08508),
+      availabilityPlan: {
+        type: 'availability-plan/time',
+        timezone: 'America/New_York',
+        entries: [
+          {dayOfWeek: 'mon', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'tue', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'wed', seats: 1, startTime: '09:00', endTime: '11:00'},
+          {dayOfWeek: 'wed', seats: 1, startTime: '12:30', endTime: '16:30'},
+          {dayOfWeek: 'thu', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'fri', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'sat', seats: 1, startTime: '10:00', endTime: '15:00'},
+        ],
+      },
+      publicData: {
+        address: {
+          city: 'New York',
+          country: 'USA',
+          state: 'NY',
+          street: '230 Hamilton Ave',
+        },
+        category: 'road',
+        gears: 22,
+        rules: 'This is a nice, bike! Please, be careful with it.',
+      },
+      price: new Money(2290, 'USD'),
+      images: [new UUID('5caede66-e4e2-4f5b-90e4-9e3f58fc7fd8')],
+    },
+    {
+      expand: true,
+      include: ['images'],
+    }
+  )
+  .then(response => {
+    // handle response
+  });
+```
+
+For more information about the plan attributes, see
+[the API documentation](https://www.sharetribe.com/api-reference/#ownlisting-availability-plan).
+By default FTW defines a day-based plan for all listings in the
+[EditListingAvailabilityPanel](https://github.com/sharetribe/flex-template-web/blob/master/src/components/EditListingAvailabilityPanel/EditListingAvailabilityPanel.js)
+component.
+
+As for creating time-based bookings, they are created just like any
+other booking. The only limitations for start and end times are that the
+end needs to be after the start and they both need to be divisible by 5
+minutes.
+
+## 3. A few things to consider
+
+### Transaction processes and listing availability plans
+
+A time-based availability plan needs to be explicitly declared for a
+listing as a default booking type can not be defined for a marketplace.
+The booking type is defined by the transaction process and a single
+marketplace can have multiple processes which can have different booking
+types. Listings are not tied to any booking process and this should be
+kept in mind when creating a booking for a listing in case your
+marketplace supports day- and time-based bookings. In this kind of
```

Comment:
It probably doesn't ever make any sense to support both day- and time-based bookings. (It's doubling the maintaining work, and time-based can handle different durations anyway: hours, days, weekends, weeks, etc.)

How about:
s/supports day- and time-based bookings/supports several different processes/